pkg/backporter/httpcache_test.go

```diff
@@ -0,0 +1,109 @@
+package backporter
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"testing"
+)
+
+//bugzillaCache tests
+
+func TestBugzillaCache(t *testing.T) {
+	testcases := []struct {
+		name      string
+		cachedKey string
+		value     []byte
+		absentKey string
+	}{
+		{
+			name:      "key 1",
+			cachedKey: "key1",
```

Comment:
Rather than having a `cachedKey`, `value` and `absentProperty` in the testcase it would be easier to just make a `cache` part of the testcase, that imediately shows whats in there and what not.